data/json/items/containers.json

```diff
@@ -271,7 +271,7 @@
         "description": "A standard plastic jug used for milk and household cleaning chemicals.",
         "price": 0,
         "weight": 190,
-        "volume": 15,
+        "volume": 18,
```

Comment:
Why eighteen volume? A gallon jug can hold 15.14 of our 250ml servings, and the lid doesn't take up that much space.